charts/barman/templates/NOTES.txt

```diff
@@ -6,9 +6,13 @@ See: http://docs.pgbarman.org/release/2.10/#setup-of-a-new-server-in-barman
 
 Initial backup:
 
+  * Get the pod name and export it
+
+    export BARMAN_POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "barman.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
```

Comment:
```suggestion
    export BARMAN_POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "barman.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o name)
```

Reply:
The default `NOTES.txt` created with a `helm create`  includes `-o jsonpath="{.items[0].metadata.name}")`. Does it really make sense to go our own way here?

Reply:
nope. You are right.